web/src/search/input/SearchPage.tsx

```diff
@@ -142,14 +145,14 @@ export class SearchPage extends React.Component<Props, State> {
 
     private onSubmit = (event: React.FormEvent<HTMLFormElement>): void => {
         event.preventDefault()
-        const query = [this.state.builderQuery, this.state.userQuery].filter(s => !!s).join(' ')
+        const query = [this.state.builderQuery, this.state.userQueryValue].filter(s => !!s).join(' ')
```

Comment:
```suggestion
        const query = [this.state.builderQuery, this.state.userQueryValue.query].filter(s => !!s).join(' ')
```

Hey @hadrian-git I noticed a bug where if you do a search from the main search page, the query will be [object Object]. Think this should fix it.